bug: some of the 0.13.5 release assets are missed on macOS 11 platform

### Summary

When installing the OpenCV library for the related plugins on macOS 11, it will re-build lots of things to take so much time, and then fail. [In this workflow](https://github.com/WasmEdge/WasmEdge/actions/runs/6744119566/job/18338593945), we can find the following error messages:

```
==> Installing opencv dependency: qt
==> Patching
==> Applying qt5-qmake-xcode15.patch
patching file qtbase/mkspecs/features/toolchain.prf
Hunk #2 succeeded at 414 (offset -1 lines).
==> python3.11 -m venv --system-site-packages --without-pip /private/tmp/qt-2023
==> python3.11 -m pip --python=/private/tmp/qt-20231103-6310-1a1kxcq/qt-everywhe
==> python3.11 -m pip --python=/private/tmp/qt-20231103-6310-1a1kxcq/qt-everywhe
==> ./configure -release -prefix /usr/local -extprefix /usr/local/Cellar/qt/6.5.
==> cmake --build .
Last 15 lines from /Users/runner/Library/Logs/Homebrew/qt/05.cmake:
[12261/14872] cd /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning && /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/share/qt/libexec/cmake_automoc_parser --cmake-autogen-cache-file /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/CMakeFiles/Positioning_autogen.dir/ParseCache.txt --cmake-autogen-info-file /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/CMakeFiles/Positioning_autogen.dir/AutogenInfo.json --output-file-path /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/Positioning_json_file_list.txt --timestamp-file-path /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/Positioning_json_file_list.txt.timestamp --cmake-autogen-include-dir-path /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/Positioning_autogen/include
[12262/14872] cd /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning && /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/share/qt/libexec/moc -o /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/qt6positioning_release_metatypes.json.gen --collect-json @/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/Positioning_json_file_list.txt && /usr/local/Cellar/cmake/3.27.7/bin/cmake -E copy_if_different /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/qt6positioning_release_metatypes.json.gen /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/meta_types/qt6positioning_release_metatypes.json
[12263/14872] /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DPROTOBUF_USE_DLLS -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon -DNDEBUG -O2 -std=gnu++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -Wall -Wextra -MD -MT qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o -MF qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o.d -o qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon/generatorbase.cpp
FAILED: qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o 
/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DPROTOBUF_USE_DLLS -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon -DNDEBUG -O2 -std=gnu++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -Wall -Wextra -MD -MT qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o -MF qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o.d -o qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/generatorbase.cpp.o -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon/generatorbase.cpp
/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon/generatorbase.cpp:8:10: fatal error: 'google/protobuf/stubs/logging.h' file not found
#include <google/protobuf/stubs/logging.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[12264/14872] /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DPROTOBUF_USE_DLLS -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon -DNDEBUG -O2 -std=gnu++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -Wall -Wextra -MD -MT qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/baseprinter.cpp.o -MF qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/baseprinter.cpp.o.d -o qtgrpc/src/tools/qtprotoccommon/CMakeFiles/QtProtocCommon.dir/baseprinter.cpp.o -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/tools/qtprotoccommon/baseprinter.cpp
[12265/14872] /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DGL_SILENCE_DEPRECATION -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_WEBCHANNEL_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_MOC_COMPAT -DQT_NETWORK_LIB -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_EXCEPTIONS -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_QMLINTEGRATION_LIB -DQT_QML_LIB -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -DWebChannel_EXPORTS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtwebchannel/src/webchannel/WebChannel_autogen/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtWebChannel -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtWebChannel.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtwebchannel/src/webchannel -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtWebChannel/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtWebChannel/6.5.2/QtWebChannel -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtCore.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/mkspecs/macx-clang -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/src/corelib -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2/QtCore -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtQml -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtQml.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtQmlIntegration -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtdeclarative/src/qmlintegration -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtNetwork -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtNetwork.framework/Headers -F/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib -DNDEBUG -O2 -std=c++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -fPIC -fapplication-extension -Winvalid-pch -Xclang -emit-pch -Xclang -include -Xclang /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtwebchannel/src/webchannel/CMakeFiles/WebChannel.dir/cmake_pch.hxx -x c++-header -MD -MT qtwebchannel/src/webchannel/CMakeFiles/WebChannel.dir/cmake_pch.hxx.pch -MF qtwebchannel/src/webchannel/CMakeFiles/WebChannel.dir/cmake_pch.hxx.pch.d -o qtwebchannel/src/webchannel/CMakeFiles/WebChannel.dir/cmake_pch.hxx.pch -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtwebchannel/src/webchannel/CMakeFiles/WebChannel.dir/cmake_pch.hxx.cxx
[12266/14872] /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DGL_SILENCE_DEPRECATION -DProtobuf_EXPORTS -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_PROTOBUF_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_EXCEPTIONS -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/protobuf/Protobuf_autogen/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtProtobuf -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtProtobuf.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/protobuf -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtProtobuf/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtProtobuf/6.5.2/QtProtobuf -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtCore.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/mkspecs/macx-clang -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/src/corelib -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2/QtCore -F/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib -DNDEBUG -O2 -std=c++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -fPIC -fapplication-extension -Winvalid-pch -Xclang -emit-pch -Xclang -include -Xclang /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/protobuf/CMakeFiles/Protobuf.dir/cmake_pch.hxx -x c++-header -MD -MT qtgrpc/src/protobuf/CMakeFiles/Protobuf.dir/cmake_pch.hxx.pch -MF qtgrpc/src/protobuf/CMakeFiles/Protobuf.dir/cmake_pch.hxx.pch.d -o qtgrpc/src/protobuf/CMakeFiles/Protobuf.dir/cmake_pch.hxx.pch -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtgrpc/src/protobuf/CMakeFiles/Protobuf.dir/cmake_pch.hxx.cxx
[12267/14872] /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -DGL_SILENCE_DEPRECATION -DPositioning_EXPORTS -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_POSITIONING_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_EXCEPTIONS -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/Positioning_autogen/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtPositioning -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtPositioning.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/../3rdparty/clip2tri -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/../3rdparty/clipper -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtPositioning/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtPositioning/6.5.2/QtPositioning -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib/QtCore.framework/Headers -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/mkspecs/macx-clang -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/src/corelib -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2 -I/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/include/QtCore/6.5.2/QtCore -F/tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtbase/lib -DNDEBUG -O2 -std=c++17 -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=11.0 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -fPIC -fapplication-extension -Winvalid-pch -Xclang -emit-pch -Xclang -include -Xclang /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/CMakeFiles/Positioning.dir/cmake_pch.hxx -x c++-header -MD -MT qtpositioning/src/positioning/CMakeFiles/Positioning.dir/cmake_pch.hxx.pch -MF qtpositioning/src/positioning/CMakeFiles/Positioning.dir/cmake_pch.hxx.pch.d -o qtpositioning/src/positioning/CMakeFiles/Positioning.dir/cmake_pch.hxx.pch -c /tmp/qt-20231103-6310-1a1kxcq/qt-everywhere-src-6.5.2/qtpositioning/src/positioning/CMakeFiles/Positioning.dir/cmake_pch.hxx.cxx
ninja: build stopped: subcommand failed.


Do not report this issue to Homebrew/brew or Homebrew/homebrew-core!


Error: You are using macOS 11.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.

Error: Process completed with exit code 1.
```

### Current State

The 0.13.5 release workflow on macOS 11 failed.

### Expected State

It should succeed and produce the corresponding release assets.

### Reproduction steps

```bash
The workflow triggers the failure. Check [the workflow](https://github.com/WasmEdge/WasmEdge/actions/runs/6744119566/job/18338593945) directly.
```


### Screenshots

```bash
![DESCRIPTION](LINK.png)
```


### Any logs you want to share for showing the specific issue

_No response_

### Components

Others

### WasmEdge Version or Commit you used

0.13.5

### Operating system information

macOS 11

### Hardware Architecture

x86_64

### Compiler flags and options

_No response_

--- a/.github/workflows/README.md
+++ b/.github/workflows/README.md
@@ -23,11 +23,6 @@ flowchart LR
 ```json
 [
   {
-    "name": "MacOS 11 (x86_64)",
-    "runner": "macos-11",
-    "darwin_version": 20
-  },
-  {
     "name": "MacOS 12 (x86_64)",
     "runner": "macos-12",
     "darwin_version": 21

--- a/.github/workflows/build-extensions.yml
+++ b/.github/workflows/build-extensions.yml
@@ -488,12 +488,6 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: Plugins_MacOS_11
-            system: MacOS 11
-            host_runner: macos-11
-            darwin_version: darwin_20
-            build_type: Release
-            arch: x86_64
           - name: Plugins_MacOS_12
             system: MacOS 12
             host_runner: macos-12

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -75,8 +75,7 @@ jobs:
     uses: ./.github/workflows/reusable-build-on-macos.yml
     with:
       version: ${{ needs.get_version.outputs.version }}
-      matrix: "[{'name':'MacOS 11 (x86_64)','runner':'macos-11','darwin_version':20,'arch':'x86_64'},
-                {'name':'MacOS 12 (x86_64)','runner':'macos-12','darwin_version':21,'arch':'x86_64'},
+      matrix: "[{'name':'MacOS 12 (x86_64)','runner':'macos-12','darwin_version':21,'arch':'x86_64'},
                 {'name':'MacOS 13 (arm64)','runner':'mac-arm64','darwin_version':22,'arch':'arm64'}]"
 
   build_on_manylinux_2014:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -53,7 +53,7 @@ jobs:
     with:
       version: ${{ needs.create_release.outputs.version }}
       matrix:
-        "[{'name':'MacOS 11 (x86_64)','runner':'macos-11','darwin_version':20,'arch':'x86_64'},
+        "[{'name':'MacOS 12 (x86_64)','runner':'macos-12','darwin_version':21,'arch':'x86_64'},
         {'name':'MacOS 13 (arm64)','runner':'mac-arm64','darwin_version':22,'arch':'arm64'}]"
       release: true
     secrets: inherit
@@ -542,10 +542,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: Plugins_MacOS_11
-            system: MacOS 11
-            host_runner: macos-11
-            darwin_version: darwin_20
+          - name: Plugins_MacOS_12
+            system: MacOS 12
+            host_runner: macos-12
+            darwin_version: darwin_21
             arch: x86_64
           - name: Plugins_MacOS_arm64
             system: MacOS 13 (arm64)

